templates/log4j2.properties.j2

```diff
@@ -1,85 +0,0 @@
-status = error
```

Comment:
Is the log4j configuration in the latest 6.x and 7.x releases still the same? Or will this result in changes for some users? 

This is a bit of a tricky change when it comes to backwards compatibility. Deploying the new ansible playbook version to an existing cluster will result in no change since this file becomes un-managed. However if a node is replaced, or a new one is added that single node will end up with a different logging configuration from the rest of the cluster.

Reply:
The log4j has a lot of changes in 6.x then 7.x versions compared to what was existing in the template we provided.

Indeed, if this file become unmanaged, existing users will keep their old versions and could have inconsistencies in log config, if they deploy new nodes. We could keep managing this file like the other to force it's update for upgrade scenarios.

However, managing these configuration files instead of reusing the ones provided by default packages increase the risk of having deprecated versions as the default files seem to change quiet regularly.